engine-src/Game/LambdaHack/Client/UI/HandleHumanGlobalM.hs

```diff
@@ -927,6 +927,45 @@ applyItem (fromCStore, (iid, (itemFull, kit))) = do
         modifySession $ \sess -> sess {sitemSel = Nothing}
         failWith "never mind"
 
+-- * CloseDir 
+
+-- | Close nearby open tile. Ask for direction, if there is more than one.
+closeDirHuman :: MonadClientUI m
+              => m (FailOrCmd RequestTimed)
+closeDirHuman = do
+  COps{cotile} <- getsState scops
+  leader <- getLeaderUI
+  b <- getsState $ getActorBody leader
+  lvl <- getLevel $ blid b
+  let vPts = vicinityUnsafe $ bpos b 
+      openPts = filter (Tile.isClosable cotile . at lvl) vPts
+      closePick p = if p `elem` openPts
+        then do
+          let s = MU.Text . T.unwords . tail . T.words . TK.tname 
+                  $ okind cotile $ lvl `at` p -- ommit first word in tname
+              dir = MU.Text $ compassText $ p `vectorToFrom` bpos b
+          msgAdd MsgDone $ makeSentence ["you", "close", dir, s]
```

Comment:
I wonder which is better, the current text, or with extra "the" or `"You close the" <+> s <+> dir <> "."`

Reply:
The last one looks the most appealing IMO. In nethack there's simply "The door closes" :-D

Reply:
Go for it. And pick either `makeSentence` or `<+.>` --- in this case either works, because subject is always "you" and so the verb does not need to be conjugated.